doc/doxygen/src/community-processes.md

```diff
@@ -0,0 +1,90 @@
+Aside of code contributions, you can also contribute to RIOT on other aspects,
+e.g. by actively participating in technical and non-technical discussions within
+the community. Regular and interim virtual meetings are announced on the
+[forum](https://forum.riot-os.org/c/community/events).
+
+## Maintainers
+
+Among contributors, some have maintainer status, which consists in rights (merge rights)
+and duties (code review duties).
+
+Maintainers can propose to give maintainer status to contributors that have been
+noticed as particularly active in some domain of RIOT. The decision to grant this
+status is then taken via consensus among maintainers. If there is consensus on
+granting the status to a particular contributor, a maintainer will contact personally
+this contributor to propose the status, which the contributor can then accept
+(or turn down).
+
+We are constantly looking for more maintainers. So if you are up for that,
+please start (or continue) contributing code and reviews!
+
+To contact maintainers, the best is to interact over actual RIOT code on
+[GitHub](https://github.com/RIOT-OS/RIOT/pulls).
+
+## Coordinators
+
+Among contributors, some are also interested in discussing RIOT matters and
+perspectives, beyond coding RIOT. In short, maintainers focus on technical
+aspects, while coordinators focus on non-technical aspects of RIOT activity.
+Of course, there is a strong overlap between maintainers and coordinators.
+
+Coordinator status consists essentially in duties (bringing up topics, debates
+and deal with the overhead). But sometimes, coordinator status does bring
+some moral reward (e.g. when an initiative brings organizational improvements,
+in the end ;).
+
+Coordinators can propose to give coordinator status to contributors that have
+been noticed as particularly active in this domain. The decision to grant this
+status is then taken via consensus among coordinators. If there is consensus
+on granting the status to a particular contributor, a coordinator will contact
+personally this contributor to propose the status, which the contributor can
+then accept (or turn down).
+
+To contact coordinators, the best is to email the genericmailing list
+riot@riot-os.org
+
+## Task Forces
+
+Parts of the community are gathered in *Task Forces*, which provide a specific
+venue focusing on a particular technical topic.
+
+*Task Forces* are (typically short-lived) efforts to achieve a well-defined goal.
+This goal might be either designing a concept and/or implementing such a concept
+or improving the quality of a certain feature/property of RIOT.
+
+Each Task Force consists of a small number of RIOT developers and has its own Wiki
+page. One or two shepherds are identified per task force as the main contact(s)
+for this activity.
+
+**Creating a new Task Force**. If you wish to launch a new task force, go ahead,
+create your wikipage, name your shepherd(s), and signal it on the forum. RIOT
```

Comment:
Question:
Do we want to keep the wiki for this ? Do we have alternative ?

Reply:
I agree that we should reconsider this. Also the reference to the mailing list.

IMO we should however rather get this in without changes other than typo fixing and do any updates as follow-up PRs. That would IMO make the git history cleaner, if this is basically just an import.